playbooks/rhv/inventory.example

```diff
@@ -0,0 +1,61 @@
+[all:vars]
+openshift_rhv_dns_zone=  # example.com
+
```

Comment:
I checked the default for openshift_master_identity_providers
https://docs.openshift.com/container-platform/3.9/install_config/configuring_authentication.html#identity-providers-ansible
and its Deny All.

It means that after OpenShift installation user will not be able to login to OpenShift?

Reply:
cluster:admin will still work from the master nodes, but no, the web interface requires some authentication setup in order to allow users in.

The goal of this PR is not to re-home the entire reference architecture in openshift-ansible, but to provide the relevant oVirt/RHV bits to help users install OCP on top of oVirt/RHV. Authentication documentation is better handled (and examples provided) elsewhere.